project/Dependencies.scala

```diff
@@ -83,15 +87,13 @@ object Dependencies {
     "com.novocode" % "junit-interface" % versions.junitInterface % "test",
     "org.scalacheck" %% "scalacheck" % versions.scalaCheckVersion % "test",
     "com.github.alexarchambault" %% "scalacheck-shapeless_1.13" % versions.scalaCheckShapelessVersion % "test"
-  ) ++ esDependencies ++ awsDependencies ++ akkaDependencies ++ dynamoDependencies ++ jacksonDependencies ++ circeDependencies
+  ) ++ esDependencies ++ awsDependencies ++ akkaDependencies ++ dynamoDependencies ++ jacksonDependencies ++ circeDependencies ++ swaggerDependencies
```

Comment:
Might be an idea to split out display into it's own package.

Reply:
So I think we should do this; next question, how.

How would you feel about an `sbt_common` directory in the top-level, which contains sub-projects for these separate packages. For example:

* sbt_common/display_models
* sbt_common/elasticsearch
* sbt_common/sierra_models

which serves as a common subdirectory for our sbt projects (and then lets us throw them into our Make ecosystem).